normally, these versions are filled in from cvs

as we have just one tarball, replace that with
a static string, filled in by a sed in prepare()

diff --git a/opentracker/ot_stats.c b/opentracker/ot_stats.c
index 7fc22de..e8f41a0 100644
--- a/opentracker/ot_stats.c
+++ b/opentracker/ot_stats.c
@@ -563,10 +563,7 @@ extern const char
 *g_version_scan_urlencoded_query_c, *g_version_trackerlogic_c, *g_version_livesync_c, *g_version_rijndael_c;
 
 size_t stats_return_tracker_version( char *reply ) {
-  return sprintf( reply, "%s%s%s%s%s%s%s%s%s%s%s%s%s%s",
-                 g_version_opentracker_c, g_version_accesslist_c, g_version_clean_c, g_version_fullscrape_c, g_version_http_c,
-                 g_version_iovec_c, g_version_mutex_c, g_version_stats_c, g_version_udp_c, g_version_vector_c,
-                 g_version_scan_urlencoded_query_c, g_version_trackerlogic_c, g_version_livesync_c, g_version_rijndael_c );
+	return sprintf( reply, "%s", "%pkgver%\n" );
 }
 
 size_t return_stats_for_tracker( char *reply, int mode, int format ) {
